hq/markdown/snyk.md

```diff
@@ -185,11 +252,14 @@ The worst case scenario is that the library is still vulnerable, has no alternat
 may wish to build and release anyway, ideally reporting the situation to a risk register. This is most likely to happen when
 a new vulnerability is discovered and the library publisher has not had chance to respond.
 
-This is cleanly achieved by creating a `.snyk` file which can act both as an exemption and the risk register itself.
+For private repositories, this is cleanly achieved by creating a `.snyk` file which can act both as an exemption and the risk register itself.
 Please try to give meaningful reasons for allowing the exemption.
 
 The `.snyk` file can then be added to the repository and the build should continue.
 
+__However, for public repositories, this would mean that the vulnerability is effectively advertised right in
```

Comment:
Do we care about this? The information is public anyway and if it means teams are more aware of it (and fix it more quickly) that is arguably better?

Be worth discussing this in the team, and maybe with InfoSec as well.

Reply:
Toned down.